bin/compile

```diff
@@ -277,9 +266,9 @@ cat <<-EOF >"${profile_script}"
 	export PATH="\$SBT_HOME/bin:\$PATH"
 EOF
 
-# Move compiled app back to where Heroku expects it
+# Remove symlink if we created one
```

Comment:
(background only) For Python and PHP we don't remove the symlinks after, in case later buildpacks happen to call those paths. I imagine that's less likely to happen for Scala though.